File: Pwa/src/app/models/viewModel/indicatorGetViewModel.ts
import { ColumnDataType } from 'src/app/_enums/column-data-type';
import { listItemViewModel } from './listItemViewModel';

export class indicatorGetViewModel{
    entityDynamicColumnId:number;
    isMultiValued:boolean;
    ColumnName:string;
    columnNameInBangla:string;
    columnDataType:ColumnDataType;
    listObjectId:number;
    listObjectName:string;
    listItem:listItemViewModel[];
    dataCollectionDate: string;
    values: string[];
}

File: Api/src/UnicefEducationMIS.Core/ViewModel/IndicatorGetViewModel.cs
using System;
using System.Collections.Generic;
using System.Text;
using UnicefEducationMIS.Core.Models;
using UnicefEducationMIS.Core.ValueObjects;

namespace UnicefEducationMIS.Core.ViewModel
{
    public class IndicatorGetViewModel
    {
       // public long Id { get; set; }
        public long EntityDynamicColumnId { get; set; }
        public int ColumnOrder { get; set; }
        public string ColumnName { get; set; }
        public string ColumnNameInBangla { get; set; }
        public bool? IsMultiValued { get; set; }
        public ColumnDataType ColumnDataType { get; set; }

        public long? ListObjectId { get; set; }
        public string ListObjectName { get; set; }

        public ICollection<ListItemViewModel> ListItems { get; set; }
        public DateTime DataCollectionDate { get; set; }

        public List<string> Values { get; set; }
        public CollectionStatus? Status { get; set; }

    }
}
